tasks/main.yml

```diff
@@ -10,11 +10,6 @@
     name: apt-transport-https
   when: ansible_pkg_mgr == "apt"
 
-- name: Add repos
-  apt_repository:
-    repo: "ppa:nginx/stable"
```

Comment:
Is this the case for all versions? Won't removing this break other things, I mean there was a reason why this PPA was chosen in the first place.
What about other nginx deployments on old (16.04) and new (18.04) Ubuntu VMs?

Reply:
we upgraded to the ppa because we wanted nginx 1.10 since ubuntu 16.04 thats the default so we have no need to upgrade any futher

Reply:
okay, don't forget to delete the sources file on the machines where we deployed the PPA.